src/constants/ssm-parameter-paths.ts

```diff
@@ -29,7 +29,8 @@ export const SSM_PARAMETER_PATHS: Record<string, SsmParameterPath> = {
   },
   PrimaryVpcId: {
     path: `${VPC_SSM_PARAMETER_PREFIX}/primary/id`,
-    description: "Virtual Private Cloud to run EC2 instances within",
+    description:
+      "Virtual Private Cloud to run EC2 instances within. Should NOT be the account default VPC. Declare additional VPCs under e.g./secondary/id and /tertiary/id and similar for subnet-related paths.",
```

Comment:
I'm not opposed to this suggestion, but I don't think it's always what we've done in accounts with more than one VPC so far. My memory is a little hazy, but I seem to recall that some accounts have two VPCs which are roughly equivalent in terms of importance (e.g. a `subscriptions` and `membership` VPC) and in this context it didn't feel appropriate to call one primary and the other secondary.

Reply:
That's helpful thanks. I'll remove that bit for now.